packages/scss/src/_icons.scss

```diff
@@ -1,4 +1,4 @@
-$font-path: _theme("icons.font-path");
+$font-path: _theme("icons.font-path") !default;
```

Comment:
As you should override the path in the variable map rather than defining it as a variable in your scss files, the "!default" shouldn't be necessary here.